### PR TITLE
fix some spelling

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: test/xunit.analyzers.tests/AssertRegexMatchShouldNotUseBoolLiteralCheckTests.cs [hunk 1]
```diff
@@ -34,7 +34,7 @@ public async void FindsWarning_ForStaticRegexIsMatch(string method)
 
         [Theory]
         [MemberData(nameof(Methods))]
-        public async void FindsWarning_ForInstaceRegexIsMatchWithInlineConstructedRegex(string method)
+        public async void FindsWarning_ForInstanceRegexIsMatchWithInlineConstructedRegex(string method)
```
Comment (sharwell) [inline test/xunit.analyzers.tests/AssertRegexMatchShouldNotUseBoolLiteralCheckTests.cs:25]: 📝 I would prefer to wait until #122 is merged to include the test changes.

Comment (SimonCropp) [reply test/xunit.analyzers.tests/AssertRegexMatchShouldNotUseBoolLiteralCheckTests.cs:25]: @sharwell no worries. let me know when u want it re-submitted

Comment (sharwell) [reply test/xunit.analyzers.tests/AssertRegexMatchShouldNotUseBoolLiteralCheckTests.cs:25]: My change is now merged 👍

**Review by sharwell — APPROVED** [review]

**Comment by bradwilson** [comment]
Thanks!

**Pull request merged**